src/lavinmq/exchange/headers.cr

```diff
@@ -2,8 +2,8 @@ require "./exchange"
 
 module LavinMQ
   class HeadersExchange < Exchange
-    def type : String
-      "headers"
+    @bindings = Hash(AMQP::Table, Set(Destination)).new do |h, k|
```

Comment:
Maybe not in this PR but maybe we should do a performance comparison when the key is a Hash instead of a amqp::table